### ISSUE TITLE
Help: Multi Monitors Set-Up

### ISSUE THREAD

**Issue by dedguy21** [ifreund/river#230]
How do I set river for dual monitors so that "workspace"

1 3 5 show up on DP-4
2 4 6 show up on DP-5

Right now only the first monitor (DP-4) can spawn an instance of a window tile, and the other can't do anything but hold wallpaper. 

So even though both monitors are recognized, using wdisplay, the second monitor isn't activated for use by river. I'm not sure through reading the Man pages how to set it for use

Obviously it's going to be through riverctl, but what is the script?

Thanks for any assistance.

**Comment by dedguy21** [comment]
Also as an aside, waybar takes a significant time to spawn when entering river, is it best not to embed in the river init, and put it in an external "autostart" file?

**Comment by Leon-Plickat** [comment]
> How do I set river for dual monitors so that "workspace"
>
>  1 3 5 show up on DP-4
>  2 4 6 show up on DP-5

You don't, because river tags do not work like that.

> Right now only the first monitor (DP-4) can spawn an instance of a window tile, and the other can't do anything but hold wallpaper.

You have to focus the other outputs if you want to use them. `riverctl focus-output next`

**Comment by dedguy21** [comment]
Thanks, in the Man pages it says you can set variable, can I set a variable for the two monitors.

**Comment by dedguy21** [comment]
ok, finally got the concept of how this window manager works, and yes, this is next level sh!t.  Very impressed. I love how the layers are working. 

I know it's not in scope, but some "rounded corners" and this probably becomes the next "meme windown manager" for r/unixporn.  

Keep up the good work.

**Comment by Leon-Plickat** [comment]
> I know it's not in scope, but some "rounded corners" and this probably becomes the next "meme windows manager" for r/unixporn.

We already had that discussion, see #81.

tl;dr: Probably not going to happen. However you can have rounded corners if they are client side. Some GTK themes have rounded corners, which will work fine with river.

**Comment by 0reoKing** [comment]
> 
> 
> > I know it's not in scope, but some "rounded corners" and this probably becomes the next "meme windows manager" for r/unixporn.
> 
> We already had that discussion, see #81.
> 
> tl;dr: Probably not going to happen. However you can have rounded corners if they are client side. Some GTK themes have rounded corners, which will work fine with river.

Is there a way for round corners on kitty and alacrity then

**Comment by ifreund** [comment]
> Is there a way for round corners on kitty and alacrity then?

Implement it in kitty and alacritty
